### PR TITLE
fix(android/oem): Check keyboard selected  before allowing setup

### LINKED ISSUE #3922: bug(oem/fv): No keyboards installed. Reverting to fallback
The Sentry log is filling up with a lot of these issues. Functionally, the FV app will just "fallback" to sil_euro_latin.

Even better would be disabling **2. Setup** until the user has done  **1. Select Keyboard**

Sentry Issue: [FV-ANDROID-H](https://sentry.keyman.com/organizations/keyman/issues/588/?referrer=github_integration)

```
No keyboards installed. Reverting to fallback
```

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #3922

Currently, the FV UX allows a user to set FV as a system keyboard before a keyboard has been selected.
This was causing a lot of Sentry issues as the app had to fallback to the sil_euro_latin keyboard.

This PR adds disables the "Setup" button until a keyboard is selected.

### Updated screenshots (final)
CSS updated to fix button and text layout

**2 Setup** button disabled when a language isn't selected
Screenshot_1606349630

**2 Setup** button enabled when a language has been selected
Screenshot_1606349642

**2 checkbox** after setup complete
Screenshot_1606349671

### CODE REVIEW

**Comment by mcdurdin** [comment]
I think I'd prefer that the Setup step is disabled visually and functionally if the first step is not completed?

Seems to be something wrong with the CSS as well -- those digits are not supposed to be hanging out of the circles.

**Comment by mcdurdin** [comment]
That workflow is nice now 💯 . Can you fixup the text in the buttons (I don't think they need to be ALL CAPS and the label isn't centred vertically) 😄

File: oem/firstvoices/android/app/src/main/assets/setup/main.html [hunk 1]
```diff
@@ -1,61 +1,77 @@
...
+                    checkbox1.style.opacity = 0.25;
+                }
+
+                disableButton2();
+            }
+
+            function disableButton2() {
+                var button2 = document.getElementById("button2");
+                button2.disabled = true;
+                button2.style.opacity = 0.25;
```
Comment (mcdurdin) [inline oem/firstvoices/android/app/src/main/assets/setup/main.html:41]: We should be able to do that with a CSS selector, e.g. `#button2:disabled { opacity: 0.25 }`

File: oem/firstvoices/android/app/src/main/assets/setup/main.html [hunk 2]
```diff
@@ -1,61 +1,77 @@
...
+            }
+
+            function enableButton2() {
+                var button2 = document.getElementById("button2");
+                button2.disabled = false;
+                button2.style.opacity = 1.0;
```
Comment (mcdurdin) [inline oem/firstvoices/android/app/src/main/assets/setup/main.html:47]: Once we have a CSS selector, this line should not be required.

File: oem/firstvoices/android/app/src/main/assets/setup/main.html [hunk 3]
```diff
@@ -1,61 +1,77 @@
...
+                }
+
+                enableButton2();
+            }
+
+            function setCheckBox1Off() {
+                var checkbox1 = document.getElementById("checkbox1");
+                if (checkbox1.src != "check-off.png") {
+                    checkbox1.src = "check-off.png";
+                    checkbox1.style.opacity = 0.25;
```
Comment (mcdurdin) [inline oem/firstvoices/android/app/src/main/assets/setup/main.html:32]: Perhaps we should tweak these to use CSS selectors as well just to tidy up.

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: oem/firstvoices/android/app/src/main/assets/setup/main.html (hunk 3 above)
Comment (darcywong00) [reply oem/firstvoices/android/app/src/main/assets/setup/main.html]: I've moved opacity to a new CSS class. TODO on getting the checkbox icon sizes correct

Comment (darcywong00) [reply oem/firstvoices/android/app/src/main/assets/setup/main.html]: Thanks for fixing that. I've updated the screenshots in the PR description.

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**